runtimes/peregrine/src/lib.rs

```diff
@@ -936,29 +936,27 @@ construct_runtime! {
 		Authorship: pallet_authorship::{Pallet, Storage} = 20,
 		AuraExt: cumulus_pallet_aura_ext = 24,
 
-		// Governance stuff
 		Democracy: pallet_democracy = 30,
 		Council: pallet_collective::<Instance1> = 31,
 		TechnicalCommittee: pallet_collective::<Instance2> = 32,
-		// placeholder: parachain council election = 33,
+		// Placeholder: parachain council election = 33,
 		TechnicalMembership: pallet_membership::<Instance1> = 34,
 		Treasury: pallet_treasury = 35,
 		// DELETED: RelayMigration: pallet_relay_migration::{Pallet, Call, Storage, Event<T>} = 36,
 		// DELETED: DynFilter: pallet_dyn_filter = 37,
 
-		// Utility module.
+		// A stateless module with helpers, to allow transmitting batch of calls
```

Comment:
I think we can get rid of this comment altogether. Runtime developers know what this module does.

Reply:
I think i like it. 😁 Might also be interesting for not so experienced devs.
 
```suggestion
		// A stateless pallet with helper extrinsics (batch extrinsics, send from different origins, ...)
```